#Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2022/10
#python > 3.7: prevent comple error future feature annotations is not defined
diff -ru psygnal-0.3.5/psygnal/_evented_model.py psygnal-0.3.5_python3.8/psygnal/_evented_model.py
--- psygnal-0.3.5/psygnal/_evented_model.py	2022-05-26 02:03:19.000000000 +0200
+++ psygnal-0.3.5_python3.8/psygnal/_evented_model.py	2022-06-23 10:41:46.536586585 +0200
@@ -1,5 +1,4 @@
-from __future__ import annotations
-
+import annotations
 import operator
 import sys
 import warnings
